docs/welcome.md

```diff
@@ -13,20 +13,21 @@ ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
 
 # Welcome to .NET
 
-See ["Getting Started with .NET Core"](core/getting-started.md) to learn how to create a simple .NET Core app. 
+See [Getting Started with .NET Core](core/getting-started.md) to learn how you can create .NET Core apps.
 
-You can build many types of apps with .NET, such as cloud, IoT and games. Your app can run on Windows, Linux, Android, macOS, and iOS. There are free tools that you can use to build apps or games. You can deploy them on servers or desktops or publish them to app stores. It's accessible to students.
+Build many types of apps with .NET, such as cloud, IoT, and games using free cross-platform tools. Your apps can run on Android, iOS, Linux, macOS, and Windows. Deploy apps to servers or desktops and publish to app stores for deployment on mobile devices. .NET is entirely accessible to students and hobbyists, and all are welcome to participate in a lively international developer community and make direct contributions to many of the .NET technologies.
```

Comment:
We could also break that sentence, but I don't really think it's too long. If we break it, it will look like this ...

> .NET is accessible to students and hobbyists. All are welcome to participate in a lively international developer community and make direct contributions to many of the .NET technologies.